### PR TITLE
docs: improve Azure DevOps setup documentation

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes:

- Reduce capitalization of page title and heading
- Fix some noun capitalization issues: npm, Yarn, Renovate
- Use Markdown to stylize text instead of using inline HTML
- Fix ordered list display so that it actually follows the numbering scheme instead of restarting at `1.` two times

## Context:

Helps with issue #7626.

While doing this work I found two further problems:

~The ordered list doesn't number through correctly, it restarts from `1.` multiple times. I'm not quite sure how to fix this. It's probably getting messed up due to the pictures. Do we even need this ordered list? I think the document order makes it clear that you should follow the instructions from top to bottom anyways?~
**Edit:** fixed by using some white-space to indent images.

This page is not findable in the Renovate docs sidebar, is this intentional? I could only find this file by using the search function on the page.

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added unit tests, or
- [ ] Unit tests + ran on a real repository

### CODE REVIEW

**Comment by viceice** [comment]
Maybe we should refactor / split the self-hosting page and include sub pages for each platform.

Then we can link from general self-hosting to platform specific docu.

File: docs/usage/setup-azure-devops.md [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
+title: Azure DevOps setup
 description: How to setup Renovate for Azure DevOps
 ---
 
-# Azure DevOps Setup (for NPM/YARN)
+# Azure DevOps setup (for npm/Yarn)
 
 1. Create a brand new pipeline within Azure DevOps, and select your source
 
 ![Azure DevOps create new pipeline](assets/images/azure-devops-setup-1.png)
```
Comment (viceice) [inline docs/usage/setup-azure-devops.md:10]: I think some idention is needed to move the image to the numbered listing

Comment (HonkingGoose) [reply docs/usage/setup-azure-devops.md]: Yes, that was the problem, it's fixed now. 😄

**Review by viceice — APPROVED** [review]

**Comment by JamieMagee** [comment]
Sorry, missed this review earlier. We're still missing a link to this page? I'm guess we need to include it in some other page?

**Pull request merged**